### PR TITLE
Make Yue's holding time/arrival rate with Erlang value

### PR DESCRIPTION
**Pull request by kbempah**
## Description
Modify Yue's simulation and modulation assumptions to be more consistent with Arash assumptions. Also changes the format of modulation assumptions from a constant value to the json format.

Fixes # (issue)

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

Change does not apply to the GUI.

## How has this change been tested?
Mostly using the existing test infrastructure and tweaking it to reflect the changes in function signature. Updated tests and fixtures can be found at:
- [x] tests/fixtures
- [x] tests/test_generate_data.py

**Test Configuration**:
* Operating System: OSX Sequioa
* Python version: Python 3.11

## Does change comply with project standards and guideline?
Standards and guidelines can be found on the [Project Homepage](https://github.com/SDNNetSim/SDON_simulator).
- [x] Commit formatting
- [x] Versioning
- [x] Code style
- [x] Coding guidelines
- [x] Unit Testing

### CODE REVIEW

**Review by ryanmccann1024 — CHANGES_REQUESTED** [review]
Really great feature addition, something we've needed for a while and you did it in a short amount of time. The only thing we need now are updated and passing tests, which we can help with if you need.

Congrats on the first major core feature addition! You're probably very familiar with the simulator input process now.

**Comment by kbempah** [comment]
Working to identify the cause of the test case failures. More so the descrepancy in local and remote test failures

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ venvs/unity_venv/venv
 
 # vscode-generated cache files
 __pycache__/
+
+json_input/run_mods/mod_formats.json
```
[the remark this answers is not in the document]
Comment (kbempah) [reply .gitignore:35]: I agree

File: arg_scripts/data_args.py [hunk 2]
```diff
@@ -1,29 +0,0 @@
-YUE_MOD_ASSUMPTIONS = {
```
[the remark this answers is not in the document]
Comment (kbempah) [reply arg_scripts/data_args.py:1]: Actually I think I deleted it. We can always reinstate it if we find we need it

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (ryanmccann1024) [reply .gitignore:35]: I think the mod_formats_path is confusing me. If we only want one file, then having a mod format path as input I don't think should be an option

File: arg_scripts/data_args.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ryanmccann1024) [reply arg_scripts/data_args.py:1]: My mistake, I thought only the contents of the file were deleted. It's good to delete the whole thing

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbempah) [reply .gitignore:35]: The thought process when writing this was to allow for the formats file to come from different sources.

[the remark this answers is not in the document]
Comment (kbempah) [reply .gitignore]: Has been set up to ignore run_mods directory.

File: tests/test_generate_data.py [hunk 3]
```diff
@@ -84,31 +84,180 @@ def test_create_pt(self, mock_file):
             check_top_dict['links'][int(link_num)] = data_dict
         self.assertEqual(topology_dict, check_top_dict)
 
-    def test_create_bw_info_yue(self):
+    @patch('builtins.open', new_callable=mock_open, read_data=json.dumps({
```
Comment (ryanmccann1024) [inline tests/test_generate_data.py:87]: Is this variable still in the file itself?

**Review by ryanmccann1024 — CHANGES_REQUESTED** [review]

File: tests/test_generate_data.py (hunk 3 above)
Comment (kbempah) [reply tests/test_generate_data.py]: It shouldn't have been. I forgot to push the changes

**Review by ryanmccann1024 — APPROVED** [review]
Great. I think it's good to go and can merge when you're ready.

**Pull request merged**